src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import { FaGithub } from 'react-icons/fa';
+import { useTranslation } from 'react-i18next';
+import { Wrapper, FooterContainer, Link, Dot, Label } from './Footer.styled';
+
+const VERSION = process.env.REACT_APP_VERSION;
+const ENV = process.env.NODE_ENV;
+const isProd = ENV === 'production';
+
+const Footer = () => {
+  const { t } = useTranslation();
+  return (
+    <Wrapper>
+      <FooterContainer>
+        {VERSION && (
+          <>
+            <Label>
+              {VERSION && `v${VERSION}`}
+              {ENV && !isProd ? `${VERSION ? ':' : null}${ENV}` : null}

```

Comment:
This logic will print out `null` if VERSION is undefined. (JS doesn't omit `null` inside backticks.) Makes sense to simplify the code here as well as its a little hard to read.

Reply:
Thanks. I don't even need that since I was checking for VERSION before. Fixed